app/posts/publish-teacher-training-courses/2023-04-20-removing-course-vacancies.md

```diff
@@ -0,0 +1,98 @@
+      href: /find-teacher-training/removing-the-school-placement-vacancy-status/
+    - text: Updating course vacancies
+      href: /publish-teacher-training-courses/updating-course-vacancies/
+screenshots:
+  items:
+    - text: Courses list
+      src: courses-list.png
+    - text: Course details with ‘Close course’ button
+      src: course-details--close.png
+    - text: Close course
+      src: close-course.png
+    - text: Course closed
+      src: course-closed.png
+    - text: Course details with ‘Open course’ button
+      src: course-details--open.png
+    - text: Open course
+      src: open-course.png
+    - text: Course opened
+      src: course-opened.png
+    - text: Withdraw open course
+      src: withdraw-open-course.png
+    - text: Withdraw closed course
+      src: withdraw-closed-course.png
+    - text: Training partners’ courses list
+      src: courses-list--training-partners.png
+---
+
+When a provider publishes a course, they can list all the schools where a candidate can gain experience.
+
+Previously, once a course was published, it was open to applications. This meant there were vacancies at some or all of the schools listed.
+
+We provided a way for users to state whether each school had vacancies.
+
+If the provider stated there were no vacancies for all schools on the list, the course is closed.  Closed courses can still be found on Find postgraduate teacher training (Find), but a candidate cannot apply.
+
+We have removed the link between the ‘Open’ and ‘Closed’ course statuses and vacancies. Users can simply state whether the course is open or closed.
+
+## What we changed
+
+We have:
+
+- removed the vacancies section
+- removed the vacancies column from the courses list
+- added closing a course
+- added opening a course
+- updated withdrawing a course
+
+We removed the ability for users to manage vacancies in Publish teacher training courses (Publish) because:
+
+- candidates do not apply to a vacancy at a school
+- providers often do not know the number of vacancies at their partner schools until late in the recruitment cycle
+- the number of vacancies can change during the recruitment cycle for reasons outside the recruitment process
+- we are providing false hope that candidates will be placed at certain schools
+- providers do not always consider a candidate’s school placement preference on the application, and some don’t know about the preference
+- the accuracy of vacancy information depends on providers frequently updating their courses
+- Apply for teacher training (Apply) is going to remove the preference for placement schools
+
+## How it works
+
+When a course is first published, it is ‘open’ on Find.
+
+In the ‘Open’ state, Publish users can:
+
+- close the course
+- view the course on Find
+- withdraw the course
+
+### Close course
+
+A provider should close a course where no placements are available.
+
+When a provider closes a course, it is still available on Find, but candidates cannot apply.
```

Comment:
Do we need to use the full service name here and on the withdraw page, as it's the first instance?

Reply:
> @Kateharries should the wording on the 'Are you sure you want to open the course' page, state the implications of the action (the reverse of what's on the close page) i.e. this will show immediately on Find and candidates will be able to apply to it?

We don't need to as the course still shows on Find even if it is closed. The text already says candidates will be able to apply if they open it.

Reply:
On any page throughout the service, if we need to refer to Publish we can just call it 'Publish' as they are already in the service.